configs/simplecopypaste/mask_rcnn_r50_fpn_syncbn-all_rpn-2conv_ssj_32x2_270k_coco.py

```diff
@@ -0,0 +1,23 @@
+_base_ = [
+    '../_base_/models/mask_rcnn_r50_fpn.py',
+    '../common/ssj_270k_coco_instance.py'
+]
+
+# For coco train2017 dataset,
```

Comment:
lines 6-8 are unnecessary., we can add a comment above line 3 to indicate it essentially trains the model by 144 epochs.